dvc/output/gs.py

```diff
@@ -1,7 +1,7 @@
-from dvc.output.s3 import S3Output
+from dvc.output.base import BaseOutput
```

Comment:
Looks like a mistake.

Reply:
@skshetry Not sure if this is needed anymore, but in the past this was done because gs shared logic with s3. Since tree refactor it might indeed no longer be needed.